doc/source/reference/ufuncs.rst

```diff
@@ -429,9 +429,9 @@ Attempting to call these methods on other ufuncs will cause a
 :exc:`ValueError`. The reduce-like methods all take an *axis* keyword
 and a *dtype* keyword, and the arrays must all have dimension >= 1.
 The *axis* keyword specifies the axis of the array over which the reduction
-will take place and may be negative, but must be an integer. The
+will take place and may be negative, in which case it counts from the last to the first axis. *axis* can be an integer, or if in :meth:`ufunc.reduce`, it can also be `None` or `tuple of ints` . The
```

Comment:
In the paragraph of "All ufuncs have four methods", I checked and it looks like only reduce can use tuple and None. Accumulate and reduceat both int only.
And I am unable to find other methods with axis as parameter in this page.
Could you tell me what are the others that you want to include here?
Thanks.